src/modifiers.rs

```diff
@@ -129,7 +131,7 @@ impl<'a> Modifier<Response> for &'a Path {
     /// Panics if there is no file at the passed-in Path.
     fn modify(self, res: &mut Response) {
         File::open(self)
-            .expect(&format!("No such file: {}", self.display()))
+            .unwrap_or_else(|_| panic!("No such file: {}", self.display()))
```

Comment:
Why is this better than `.expect(&format!("No such file: {}", self.display()))`? Is it just to spare the call to self.display?

Reply:
Probably because you don't call `format` when there is no error

Reply:
@phlmn I also don't understand this change. `format!` can be used for all sorts of use cases, I'm not sure I follow why it can't be used to build a string for an `expect` call.

Reply:
@RobertWHurst I did not write that line and I don't think there is any noticeable difference, but: If you write `expect(&format(…))` the `format(…)` is always executed while `unwrap_or_else(|_| panic!(…))` only executes a format if the `Result` is an `Err`.